.circleci/config.yml

```diff
@@ -49,32 +49,6 @@ jobs:
     - store_artifacts:
         path: junit
 
-  semantic-release:
-    executor: node10
-
-    steps:
-    - setup
-    - run:
-        name: revert changes to package-lock.json
-        command: git checkout -- package-lock.json
-
-    - helix-post-deploy/config-wsk
-
-    - helix-post-deploy/config-env
-
-    - run:
-        name: Semantic Release
-        command: npm run semantic-release
-
-    - run:
-        name: Link Sequences
-        command: npm run deploy-sequences
-
-    - helix-post-deploy/monitoring:
```

Comment:
Why did you remove the monitoring config?

Reply:
because we can no longer use it with github actions. is this needed, if the sequence number doesn't change?

Reply:
@rofe will there be a problem if we remove this ?

Reply:
@tripodsan No, I think the current monitoring setup for `@adobe/helix-status` will stay intact as is, but will no longer be updated in the future. Unless we create a dedicated `helix-runtime-status` service as you suggested [here](https://github.com/adobe/helix-status/issues/231#issuecomment-727871096)